pkg/v1/task/task.go

```diff
@@ -280,7 +279,3 @@ func (t *DurableTaskDeclaration[I]) GetName() string {
 func (t *NamedTaskImpl) GetName() string {
 	return t.Name
 }
-
-func getActionID(workflowName, taskName string) string {
-	return strings.ToLower(fmt.Sprintf("%s:%s", workflowName, taskName))
-}
```

Comment:
this too: https://github.com/hatchet-dev/hatchet/pull/1637/files#diff-adbea65fca87f25c1415af8d20c07d3891d0f7ecf39887b0ff07e56b1dc3b5f7R284-R286